pkgs/by-name/nt/ntfs2btrfs/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake, fmt, lzo, pkg-config, zlib, zstd }:
+
+stdenv.mkDerivation rec {
+  pname = "ntfs2btrfs";
+  version = "20240115";
+
+  nativeBuildInputs = [ cmake fmt lzo pkg-config zlib zstd ];
+
+  src = fetchFromGitHub {
+    owner = "maharmstone";
+    repo = pname;
```

Comment:
```suggestion
    repo = "ntfs2btrfs";
```

Reply:
for this, i think i already defined `pname` as the repo name anyway, isn't it better to do `repo = pname` so i have one less line to think about, in case the repo name (and along with it, probably the binary + package name) changes?

Reply:
It is not better to overuse pname as this can break with overlays and is just coincidence that the strings match. It is also a problem for the conversion to finalAttrs.

Reply:
ooooo i see, thanks for the input! going forward with this change.